card/src/main/java/com/adyen/checkout/card/CardViewProvider.kt

```diff
@@ -23,7 +23,7 @@ internal object CardViewProvider : ViewProvider {
         defStyleAttr: Int
     ): ComponentViewNew {
         return when (viewType) {
-            CardComponentViewType -> CardViewNew(context, attrs, defStyleAttr)
+            CardComponentViewType -> CardViewNew(context, null, defStyleAttr)
```

Comment:
Is this `null` by mistake?

Reply:
Ohhh my bad, I didn't mean to commit this... I have to do this to be able to render the view, because of this weird bug with view attribute inspection :(